.github/workflows/pages.yaml

```diff
@@ -30,27 +30,26 @@ jobs:
 
     env:
       PUBLIC_URL: https://boardwalk.swarm.city
+      NODE_VERSION: 18
 
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
 
-      - name: Use Node.js 16
+      - name: Use Node.js ${{ env.NODE_VERSION }}
         uses: actions/setup-node@v3
         with:
           cache: npm
-          node-version: 16
+          node-version: ${{ env.NODE_VERSION }}
 
       - name: Install npm deps
         run: npm ci
 
       - name: Build for gh-pages
         run: |
           npm run build
-          echo "boardwalk.swarm.city" > ./dist/CNAME
+          echo "{{ env.PUBLIC_URL }}" > ./dist/CNAME
```

Comment:
Will this work? There is now extra `https://` at the beginning

Reply:
Good catch. What was `PUBLIC_URL` good for previously?

Reply:
It is good for the build for loading assets if you are serving from subdirectory (so any import would now have the full URL instead of just `.`). Probably not needed anymore actually.